src/main/java/org/bahmni/mart/form2/translations/util/Form2TranslationsReader.java

```diff
@@ -44,8 +44,11 @@ public String getTranslation(Form2Translation form2Translation, String translati
     }
 
     private JSONObject getTranslationsAsJSONObject(String formName, int version) {
-        String translationsFilePath = translationMetadata.getTranslationsFilePath(formName, version);
-        File translationsFile = new File(translationsFilePath);
+
+        String translationsFilePathWithUuid = translationMetadata.getTranslationsFilePathWithUuid(formName, version);
+        File translationsFile = new File(translationsFilePathWithUuid);
+        if (!translationsFile.exists())
+            translationsFile = new File(translationMetadata.getTranslationsFilePath(formName, version));
         if (!translationsFile.exists())
             translationsFile = new File(translationMetadata.getNormalizedTranslationsFilePath(formName, version));
```

Comment:
Line number 53 always overrides line number 51?

Reply:
+1

Reply:
Since the condition is same in both cases and the implementation is different, Couldn't use else or else-if. And line 53 never overrides line 51. Its always, either of those will get executed.